client/my-sites/plugins/index.web.js

```diff
@@ -152,6 +153,19 @@ export default function ( router ) {
 		clientRender
 	);
 
+	router(
+		`/${ langParam }/plugins/mailpoet/confirmation/:site_id`,
```

Comment:
Likewise, I think this should be renamed to `/${ langParam }/plugins/mailpoet-business/upgrade/:site_id`

Reply:
Agreed, to be consistent with the name of the component.